smithy-swift-codegen/src/main/kotlin/software/amazon/smithy/swift/codegen/AuthSchemeResolverGenerator.kt

```diff
@@ -285,7 +285,7 @@ fun Parameter.toSymbol(): Symbol {
     val swiftType = when (type) {
         ParameterType.STRING -> SwiftTypes.String
         ParameterType.BOOLEAN -> SwiftTypes.Bool
-        else -> throw CodegenException("Unsupported parameter type: $type")
+        ParameterType.STRING_ARRAY -> SwiftTypes.StringArray
```

Comment:
Add `STRING_ARRAY` case and eliminate the `else` so this fails to compile when a new `ParameterType` is added in the future.